### DISCUSSION TITLE
Placemarker display problem

### DISCUSSION THREAD

**Discussion by NormR** [organicmaps/organicmaps#5312] [Q&A]
Placemarkers only display on the bottom part (1 inch on 7 inch screen) of the screen on my Nexus 7 Android 6 tablet.
When the view is moved to put a placemarker in the center of the screen, the placemarker disappears.  If the view is moved down, the placemarker appears again.  This also happens with Maps.me
Is there a setting I need to fix?
Thanks,
Norm

**Comment by Misalf-git** [comment]
Sounds similar to this
https://github.com/organicmaps/organicmaps/issues/1112

**Reply by NormR** [reply]
Thanks for that.
Yes, that sounds like my problem.  I had been using Maps.me for years without any problems.  Recently Maps,me started having this problem after installing a new version. I don't remember when I first saw the problem,
 In searching for a solution I found a download page for OOMHB and installed it on my tablet hoping it would fix the problem, No luck, OOMHB has the same problem.   
I have tried uninstalling Maps.me (was version 14) and installing version 12.3.3. Same problem.  
Could the maps I download have something in them that would cause the problem?

**Comment by NormR** [comment]
I dug out my old tablet: Nexus7(2012) Android 5.1.1 and downloaded and installed the current OOMHB program(2023.6.4).  It worked.  The placemarkers were shown over all the screen.  I downloaded several maps - Barcelona and Tarragona for testing.
